### PR TITLE
chore(syncing-up): prune down to community forum links

### LINKED ISSUE #792: add state sync to node start documentation
https://docs.agoric.com/guides/getting-started/syncing-up.html is somewhat onerous due to [lack of state sync support](https://github.com/Agoric/agoric-sdk/issues/3769). But that's due to change any time now: https://github.com/Agoric/agoric-sdk/pull/7225 

That PR says:
> Documentation Considerations
> Nothing special besides normal cosmos state-sync instructions.

looking around, I find:

 - https://github.com/cosmos/gaia/blob/main/docs/hub-tutorials/join-mainnet.md#state-sync
 - https://github.com/tendermint/tendermint/blob/v0.34.x/spec/p2p/messages/state-sync.md

cc @mhofman @michaelfig @Tyrosine22 @Just-a-Lorax @arirubinstein @ivanlei

### PR DESCRIPTION
**Pull request by dckc**
rendered:

 - [Syncing Up on Mainnet](https://dc-prune-syncing.documentation-7tp.pages.dev/guides/getting-started/syncing-up.html)

fixes #792

cc @kbennett2000 , JD

### CODE REVIEW

File: main/guides/getting-started/syncing-up.md [hunk 1]
```diff
@@ -1,99 +1,13 @@
 # Syncing Up on Mainnet
 
-To run a node on the `agoric-3` network from scratch, please follow the
-steps below.
+Tune in to [Network Upgrades Governance](https://community.agoric.com/c/governance/network-upgrades/15) in the [Agoric Community Forum](https://community.agoric.com/) for upcoming changes to mainnet node operations, including deployment of
```
Comment (mhofman) [inline main/guides/getting-started/syncing-up.md:3]: past and upcoming changes?

File: main/guides/getting-started/syncing-up.md [hunk 2]
```diff
@@ -1,99 +1,13 @@
...
+new [agoric-sdk software releases](https://github.com/Agoric/agoric-sdk/releases).
 
-If you wish to restore from a recent community snapshot, please follow the instructions under [Current Community Snapshot](#current-community-snapshot) below.
+Mainnet operation is supported on a peer-to-peer basis.
```
Comment (mhofman) [inline main/guides/getting-started/syncing-up.md:6]: Should there be a link to discord here?

**Review by kbennett2000 — APPROVED** [review]
This looks like much less maintenance, and much simpler!

**Review by mhofman — APPROVED** [review]
Just a couple small questions / suggestions

File: main/guides/getting-started/syncing-up.md (hunk 1 above)
Comment (dckc) [reply main/guides/getting-started/syncing-up.md]: going with just "changes", no "upcoming" qualifier

**Pull request merged**